marshmallow_objects/models.py

```diff
@@ -1,5 +1,6 @@
 import collections
 import contextlib
+import copy
```

Comment:
unnecessary import

Reply:
how so?

Reply:
I don't see a code which uses this library, also pep8 checks says that this is unused import. Please remove this line